unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/ParticlesOnAvatarVisualCue.cs

```diff
@@ -0,0 +1,41 @@
+using DCL;
+using UnityEngine;
+
+[RequireComponent(typeof(AvatarRenderer))]
+public class ParticlesOnAvatarVisualCue : MonoBehaviour
+{
+    [SerializeField] private AvatarRenderer.VisualCue avatarVisualCue;
+    [SerializeField] private GameObject particlePrefab;
+    [SerializeField] private bool followAvatar;
+
+    private AvatarRenderer avatarRenderer;
+    private void Awake()
+    {
+        avatarRenderer = GetComponent<AvatarRenderer>();
+        if (avatarRenderer == null)
+            return;
```

Comment:
As the class already have the `[RequireComponent(typeof(AvatarRenderer))]`, it is not necessary to check if the `AvatarRenderer` component exists, isn't it?

Reply:
You are completly right, thanks!